README.md

```diff
@@ -1,5 +1,77 @@
-# Core Java Apps
+# Homa Almasieh . Jarvis Consulting
 
-1. [Java Grep App](./grep)
-2. [JDBC App](./jdbc)
-3. [Twitter CLI App](./twitter)
+I recently completed Data Analytics BootCamp from the University of Toronto focused on gaining technical programming skills in Excel, VBA, Python, R, JavaScript, SQL Databases, Tableau, Big Data, and Machine Learning. I worked with various datasets during the BootCamp program and according to the content of the projects, I used various tools to analyze the data, design the model and visualize the finding through Tableau and dashboards. In addition, I am a former Assistant Professor of University in the area of Applied Mathematics with a brilliant and valuable background in the field of teaching and scientific research. With this extensive background of Mathematics and Data, I started  my training program at Jarvis by developing various projects and this mentorship helped me to gain skills in increasing code efficiency, debugging, and gaining comprehensive knowledge in Linux/Bash, Git, Java, Docker, RDBMS and Springboot. I am a self-initiated and avid learner with high attention to detail, critical thinking and logical problem-solving. My adaptability allows me to work in many different environments, in addition to speaking with people from various backgrounds. I am looking forward to speaking about my academic and professional background for developing positions mainly in the data analytics field.
```

Comment:
I recently completed Data Analytics BootCamp from the University of Toronto focused on gaining technical programming skills in Excel, VBA, Python, R, JavaScript, SQL Databases, Tableau, Big Data, and Machine Learning. I worked with various datasets during the BootCamp program and according to the content of the projects, I used various tools to analyze the data, design the model and visualize the finding through Tableau and dashboards. In addition, I am a former Assistant Professor of University in the area of Applied Mathematics with a brilliant and valuable background in the field of teaching and scientific research. With this extensive background in Mathematics and Data, I started my training program at Jarvis by developing various projects. This mentorship helped me increase code efficiency, debugging and gain comprehensive knowledge in Linux/Bash, Git, and Java Docker, RDBMS and Springboot. I am a self-initiated and avid learner with great attention to detail, critical thinking and logical problem-solving. My adaptability allows me to work in many different environments and speak with people from various backgrounds.

Reply:
Done!